code/game/objects/structures/ai_core.dm

```diff
@@ -295,8 +308,17 @@
 				if(tool.tool_behaviour == TOOL_CROWBAR && core_mmi)
 					tool.play_tool_sound(src)
 					balloon_alert(user, "removed [AI_CORE_BRAIN(core_mmi)]")
-					core_mmi.forceMove(loc)
-					return
+					if(remote_ai)
+						remote_ai.break_core_link()
+						if(!IS_MALF_AI(remote_ai))
+							//don't pull back shunted malf AIs
+							remote_ai.death(gibbed = TRUE, drop_mmi = FALSE)
```

Comment:
```suggestion
							remote_ai.death(gibbed = TRUE, drop_mmi = FALSE)
							return
```
Unless I'm reading it wrong, gibbing the AI nulls the `remote_ai` var, so trying to run the code after this line wouldn't be wise.

Reply:
Good catch. You pointing this out also made me look over the rest of the code and there's a couple of other things I need to fix. Thank you